### PR TITLE
Tracer spec globals review

### PR DESCRIPTION
**Pull request by z1c0**
Remove spec part about singleton Tracers which leads to confusion.
regarding named tracers.

### CODE REVIEW

**Comment by tsloughter** [comment]
Note I removed this in https://github.com/open-telemetry/opentelemetry-specification/pull/354

File: specification/api-tracing.md [hunk 1]
```diff
@@ -80,18 +80,18 @@ exposes methods for creating and activating new `Span`s. The `Tracer` is
...
 
 New `Tracer` instances can be created via a `TracerFactory` and its `getTracer`
 method. This method expects two string arguments:
 
+`TracerFactory`s are generally expected to be used as singletons. Implementations
+SHOULD provide a single global default `TracerFactory`.
+
+Some applications may use multiple `TracerFactory` instances, e.g. to provide
+different settings (e.g. `SpanProcessor`s) to each of those instances and -
+in further consequence - to the `Tracer` instances created by them.
```
Comment (tsloughter) [inline specification/api-tracing.md]: I didn't know this was supposed to be a feature. Is there an OTEP regarding this or is it part of the "named tracer" otep?

Comment (Oberon00) [reply specification/api-tracing.md]: This is just a logical consequence of the introduction of the tracer factory. Look at the paragraph that was removed, the "new" paragraph just replaces Tracer with TracerFactory.

Comment (tsloughter) [reply specification/api-tracing.md]: Yea, nevermind, got that cleared up in my API PR.

**Pull request merged**